web/backend/src/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.8'

```

Comment:
You should add some instructions on how to use the docker-compose.

Reply:
Should I do that using comments ? I thought it would be good to explain it in the ReadMe. What do you think ?

Reply:
Updating the README.md is a good idea.

Reply:
Okk, I'm on it !